### PR TITLE
Unit 6

### PR DESCRIPTION
**Pull request by alder154**
Pull request for unit 6

### CODE REVIEW

File: 6. Prototypes.js [hunk 1]
```diff
@@ -1,13 +1,12 @@
...
 let cat = Object.create(animal, {
-    fly: {
-        value: false,
-        }
-    }
-);
+  fly: {
+    value: false,
+  },
+});
```
Comment (rimus) [inline 6. Prototypes.js:13]: You also need to make classes inheritance here.

**Review by rimus — CHANGES_REQUESTED** [review]

File: 6. Prototypes.js (hunk 1 above)
Comment (alder154) [reply 6. Prototypes.js:13]: No mention for classes inheritance there. But'll add
Снимок экрана 2022-03-18 в 19 20 35

Comment (rimus) [reply 6. Prototypes.js:13]: Hmm. Maybe tasks were changed. Ok, i understood. Thanks.

**Review by rimus — APPROVED** [review]
Looks good.

**Pull request merged**